### PR TITLE
fix: jest configuration not being used and migrated some jest config

### PR DESCRIPTION
**Pull request by ischeng-ucsd**
Fixed jest not recognizing the jest config file and migrated configs in package.json to jest config file to keep everything in one centralized location.

### CODE REVIEW

File: jest.config.json [hunk 1]
```diff
@@ -1,4 +1,8 @@
 {
+  "moduleNameMapper": {
+    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/chromeMock.js",
```
Comment (hwanggit) [inline jest.config.json:3]: Unit tests in Linter were failing because of an edit in package.json

Comment (hwanggit) [reply jest.config.json:3]: I added it back and now it passes.

Comment (hwanggit) [reply jest.config.json:3]: So now moduleNameMapper exists in both package.json and this file. Are you ok with that?

Comment (ischeng-ucsd) [reply jest.config.json:3]: Whoops, I'll remove the jest configs in package.json

Comment (hwanggit) [reply jest.config.json:3]: Code quality is failing

**Pull request merged**